src/components/Footer.tsx

```diff
@@ -0,0 +1,123 @@
+            <div className="max-w-5xl mx-auto">
+                <div className="flex flex-col md:flex-row justify-between gap-10">
+                    <div className="flex flex-col gap-6 mb-8 md:mb-0">
+                        <Logo width={40} height={44} />
+                        <p className="w-60 text-slate-500 text-sm leading-relaxed">
+                            Takeaway & Delivery template for small - medium businesses.
+                        </p>
+                    </div>
+
+                    <div className="mb-8 md:mb-0">
+                        <h3 className="text-zinc-950 text-xs font-semibold uppercase tracking-wider mb-5">Company</h3>
+                        <ul className="flex flex-col gap-5 text-slate-500 text-sm space-y-2">
+                            {companyLinks.map((link, index) => (
+                                <li key={index}><a href={link.url}>{link.name}</a></li>
+                            ))}
+                        </ul>
+                    </div>
+
+                    <div className="mb-8 md:mb-0">
+                        <h3 className="text-zinc-950 text-xs font-semibold uppercase tracking-wider mb-5">Template</h3>
+                        <ul className="flex flex-col gap-5 text-slate-500 text-sm space-y-2">
+                            {templateLinks.map((link, index) => (
+                                <li key={index}><a href={link.url}>{link.name}</a></li>
+                            ))}
+                        </ul>
+                    </div>
+
+                    <div>
+                        <h3 className="text-zinc-950 text-xs font-semibold uppercase tracking-wider mb-5">Flowbase</h3>
+                        <ul className="flex flex-col gap-5 text-slate-500 text-sm space-y-2">
+                            {flowbaseLinks.map((link, index) => (
+                                <li key={index}><a href={link.url}>{link.name}</a></li>
+                            ))}
+                        </ul>
+                    </div>
+                </div>
+
+                <div className="border-t border-gray-300 mt-24 mb-16"></div>
+
+                <div className="flex flex-col md:flex-row justify-between items-center text-slate-500 text-sm gap-4">
+                    <div className="flex space-x-1">
+                        {credits.map((credit, index) => (
+                            <React.Fragment key={index}>
+                                <span>{credit.text}</span>
+                                <a href={credit.url} className="text-teal-500">{credit.name}</a>
+                            </React.Fragment>
+                        ))}
+                    </div>
+
+                    <div className="flex space-x-4">
+                        <div className="w-8 h-8 rounded-full border border-gray-300 flex items-center justify-center">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 18 18"
```

Comment:
It would be better to put svgs into separate file and import to components. Here and bellow)